source/_components/sensor.growatt.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "Growatt sensor"
+description: "Instructions on how to integrate a Growatt solar panel plant into Home Assistant."
+date: 2019-01-30 15:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: growatt.png
+ha_category: Energy
+ha_release: 0.88.0
+---
+
+The Growatt sensor enables you to monitor Growatt solar panel plants that have a WiFi module that send the data to [the Growatt server](http://server.growatt.com). The sensor will pull the total `kWh` and daily `kWh` from the server.
```

Comment:
Maybe https://server.growatt.com/ could be used to encourage use of https (more privacy, better security).

Reply:
Good point, it actually is `https` 😄